README.md

```diff
@@ -172,6 +172,25 @@ to generate a similar UI. Follow the
 to access the Katib UI.
 ![katibui](./docs/images/katib-ui.png)
 
+### New UI
```

Comment:
Can you re-run [`update-readme-toc`](https://github.com/kubeflow/katib/blob/master/scripts/update-readme-toc.sh) script to update table of contents, please ?

Reply:
Good catch, I'm on it

Reply:
Added a commit for this. The table of contents is like this now:
```
Web UI
  New UI
```

Let me know if you'd like any further changes on this